.circleci/base_config.yml

```diff
@@ -527,7 +527,15 @@ jobs:
               export UBSAN_OPTIONS="log_exe_name=true:log_path=`pwd`/work/alubsan.log:print_stacktrace=1:suppressions=ubsan_arangodb_suppressions.txt:print_suppressions=0:external_symbolizer_path=`pwd`/utils/llvm-symbolizer-client.py"
               export SAN_MODE=alubsan
             elif [ "<< pipeline.parameters.sanitizer >>" = "tsan" ]; then
-              export TSAN_OPTIONS="log_exe_name=true:log_path=`pwd`/work/tsan.log:detect_deadlocks=true:second_deadlock_stack=1:suppressions=tsan_arangodb_suppressions.txt:print_suppressions=0:external_symbolizer_path=`pwd`/utils/llvm-symbolizer-client.py"
+              # Needed to ignore TSAN errors from OpenMP
+              find /usr -iname libarcher.so
```

Comment:
```suggestion
```
I assume this was only needed to find it in the first place, and is now hard-coded in the OMP_TOOL_LIBRARIES export below?

Reply:
Ah yes, you are right, this can be removed